CHANGELOG.md

```diff
@@ -1,11 +1,15 @@
 # Changelog
 
-## version tbd (not published)
+## 5.0.0 (not published yet)
```

Comment:
We don't have anything broken, just a big refactor on drawer, right? Asking this to ensure that we don't need a `_breaking changes_` section

Reply:
There are breaking changes (class name changes and moving the activeItem prop to the drawer instead of the navgroup are the most breaking), hence the major version bump. The Keepachangelog format we use does not have a separate section for breaking changes

Reply:
I could potentially do a little work to allow you to continue to specify activeItem on the nav group to maintain backwards compatibility, but I would prefer not to (originally we had posed that doing it that way would allow people to have multiple items active at a time in different groups, but I don't think that's really a use case we need to support).